policy-controller/k8s/status/src/index.rs

```diff
@@ -703,24 +798,32 @@ impl Index {
                 }
                 Err(error) => {
                     self.metrics.patch_channel_full.inc();
-                    tracing::error!(%id.namespace, route = ?id.gkn, %error, "Failed to send HTTPRoute patch");
+                    tracing::error!(%id.namespace, route = ?id.gkn, %error, "Failed to send {} patch", id.gkn.kind.as_ref());
```

Comment:
we're already logging the GKN in the `route` field, so I don't think we need to also log the kind in addition.  can probably have this say something like "Failed to send route patch" or something similar.

Reply:
Fixed